designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -179,9 +212,9 @@ trait StandardRemoteEnvironment extends FailFastCirceSupport with RemoteEnvironm
     } yield validation
   }
 
-  private def saveProcess(process: DisplayableProcess, comment: UpdateProcessComment)(implicit ec: ExecutionContext): Future[Unit] = {
+  private def saveProcess(process: DisplayableProcess, comment: UpdateProcessComment, createdBy: Option[String] = None)(implicit ec: ExecutionContext): Future[Unit] = {
```

Comment:
1. Please don't use default values. Default values are acceptable in tests, in dsl etc. In normal situation they can introduce hard to diagnose bug, that someone won't pass some value on some level. It's an opposition to statical type checking 
2. Let's keep naming consistent: let's stick with `forwardedUserName`. `forwardedUserName` became `createdBy` when it is wrapped by  `Remote[$name]`